src/software/ai/hl/stp/stp_referee_command_play_selection_test.cpp

```diff
@@ -224,23 +221,55 @@ class STPRefereeCommandPlaySelectionTest
+    RefereeCommand ref_cmd = std::get<0>(GetParam());
+    Ball ball              = std::get<1>(GetParam());
 
+    world.updateRefereeCommand(ref_cmd);
+    world.updateGameStateBall(ball);
     try
     {
         auto play_ptr = stp.calculateNewPlay(world);
     }
     catch (...)
     {
-        FAIL() << "No play for game state " << GetParam();
+        FAIL() << "No play for game state " << std::get<0>(GetParam());
     }
 }
 
-auto all_referee_commands = allValuesRefereeCommand();
+// TODO (Issue #1665): Include `BALL_PLACEMENT_US` and `BALL_PLACEMENT_THEM` when ball
+// placement states have plays
 
-// TODO (Issue #1330): Reenable these tests
-// INSTANTIATE_TEST_CASE_P(AllRefereeCommands, STPRefereeCommandPlaySelectionTest,
-//                        ::testing::ValuesIn(all_referee_commands.begin(),
-//                                            all_referee_commands.end()));
+// NORMAL_START is omitted since there is no preceding PREPARE state
+INSTANTIATE_TEST_CASE_P(
+    AllRefboxGameStates, STPRefereeCommandPlaySelectionTest,
+    ::testing::Values(
+        std::make_tuple(RefereeCommand::HALT,
+                        Ball(Point(0, 0), Vector(), Timestamp::fromSeconds(0))),
+        std::make_tuple(RefereeCommand::STOP,
+                        Ball(Point(0, 0), Vector(), Timestamp::fromSeconds(0))),
+        std::make_tuple(RefereeCommand::FORCE_START,
+                        Ball(Point(0, 0), Vector(), Timestamp::fromSeconds(0))),
+        std::make_tuple(RefereeCommand::PREPARE_KICKOFF_US,
+                        Ball(Point(0, 0), Vector(1, 0), Timestamp::fromSeconds(0))),
+        std::make_tuple(RefereeCommand::PREPARE_KICKOFF_THEM,
+                        Ball(Point(0, 0), Vector(-1, 0), Timestamp::fromSeconds(0))),
+        std::make_tuple(RefereeCommand::PREPARE_PENALTY_US,
+                        Ball(Point(2.7, 0), Vector(), Timestamp::fromSeconds(0))),
+        std::make_tuple(RefereeCommand::PREPARE_PENALTY_THEM,
+                        Ball(Point(-2.7, 0), Vector(), Timestamp::fromSeconds(0))),
+        std::make_tuple(RefereeCommand::DIRECT_FREE_US,
+                        Ball(Point(-4.3, -2.8), Vector(), Timestamp::fromSeconds(0))),
+        std::make_tuple(RefereeCommand::DIRECT_FREE_THEM,
+                        Ball(Point(4.3, 2.8), Vector(), Timestamp::fromSeconds(0))),
+        std::make_tuple(RefereeCommand::INDIRECT_FREE_US,
+                        Ball(Point(-2, 2.8), Vector(), Timestamp::fromSeconds(0))),
+        std::make_tuple(RefereeCommand::INDIRECT_FREE_THEM,
+                        Ball(Point(2, -2.8), Vector(), Timestamp::fromSeconds(0))),
+        std::make_tuple(RefereeCommand::TIMEOUT_US,
+                        Ball(Point(0, 0), Vector(), Timestamp::fromSeconds(0))),
+        std::make_tuple(RefereeCommand::TIMEOUT_THEM,
+                        Ball(Point(0, 0), Vector(), Timestamp::fromSeconds(0))),
+        std::make_tuple(RefereeCommand::GOAL_US,
```

Comment:
I don't think it makes sense for `GOAL_*` states to have a selected play

Reply:
Removed `GOAL_US` and `GOAL_THEM`.